pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -1333,7 +1333,7 @@ func TestEventTransientData(t *testing.T) {
 
 		connID := generateRandomID()
 
-		err = svc.connectionStore.TransientStore().Put(eventTransientDataKey(connID), []byte("invalid data"))
+		err = svc.connectionStore.SaveEventData(connID, []byte("invalid data"))
```

Comment:
`Data` seems like an unnecessary suffix.

Reply:
it is called as `getEventTransientData/storeEventTransientData` in DID exchange service.
Should be SaveEventTransientData??

Reply:
SaveEvent?

Reply:
changed it to `SaveEvent`